src/components/landing/landing-subjects.tsx

```diff
@@ -54,19 +54,21 @@ export function LandingSubjects() {
   )
 }
 
-// TODO: needs type declaration
-type SubjectProps = any
-
-function Subject(props: SubjectProps) {
-  const { url, title, SubjectSVG } = props
+interface SubjectProps {
+  url: string
+  title: string
+  SubjectSVG: React.ReactNode
```

Comment:
should be in lowercase since `SubjectSVG` suggests this is a component (which it is not)

Reply:
It's an React.FunctionComponent no?

Reply:
Nope. `React.ReactNode` represents a rendered component, e.g. `<FooBar />`. The corresponding component would be `FooBar` (so without JSX). So from naming, this should be:
```tsx
const foo = {
  subjectSVG: <Subject /> // type React.ReactNode   or
  SubjectSVG: Subject // type React.ComponentType<{}>
}
```